README.adoc

```diff
@@ -72,12 +205,15 @@ Logging output is displayed. The service should be up and running within a few s
 
 == Test the application
 
-Now that the application is running, you can test it.
+Now that the application is running, you can test it. Open http://localhost:8080, type a message
+in the input text box, press the "Publish!" button and verify that the message was correctly logged
+in your process terminal window.
 
```

Comment:
I got an error while going through the guide, I would think it would be helpful to include a link to setup default credentials if someone wants to run this locally.

```
Caused by: java.io.IOException: The Application Default Credentials are not available. They are available if running in Google Compute Engine. Otherwise, the environment variable GOOGLE_APPLICATION_CREDENTIALS must be defined pointing to a file defining the credentials. See https://developers.google.com/accounts/docs/application-default-credentials for more information.
	at com.google.auth.oauth2.DefaultCredentialsProvider.getDefaultCredentials(DefaultCredentialsProvider.java:111) ~[google-auth-library-oauth2-http-0.7.0.jar:na]
	at com.google.auth.oauth2.GoogleCredentials.getApplicationDefault(GoogleCredentials.java:103) ~[google-auth-library-oauth2-http-0.7.0.jar:na]
	at com.google.auth.oauth2.GoogleCredentials.getApplicationDefault(GoogleCredentials.java:76) ~[google-auth-library-oauth2-http-0.7.0.jar:na]
	at com.google.api.gax.core.GoogleCredentialsProvider.getCredentials(GoogleCredentialsProvider.java:54) ~[gax-1.4.1.jar:na]
	at com.google.cloud.pubsub.v1.TopicAdminClient.<init>(TopicAdminClient.java:153) ~[google-cloud-pubsub-0.20.1-beta.jar:0.20.1-beta]
	at com.google.cloud.pubsub.v1.TopicAdminClient.create(TopicAdminClient.java:141) ~[google-cloud-pubsub-0.20.1-beta.jar:0.20.1-beta]
	at org.springframework.cloud.gcp.pubsub.autoconfig.GcpPubsubAutoConfiguration.topicAdminClient(GcpPubsubAutoConfiguration.java:133) ~[spring-cloud-gcp-starter-pubsub-1.0.0.BUILD-SNAPSHOT.jar:1.0.0.BUILD-SNAPSHOT]
	... 43 common frames omitted

```